.github/workflows/format.yml

```
@@ -0,0 +1,57 @@
+on:
+  pull_request_review:
+    types: [submitted]
+
+jobs:
+  should_run_format:
+    runs-on: ubuntu-latest
+    if: github.event.review.state == 'approved'
+    outputs:
+      run_format: ${{ steps.check.outputs.run_format }}
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.8)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)